### PR TITLE
Fix "Copy URL" functionality

### PR DESCRIPTION
**Pull request by AeroCross**
Hey mate!

Thanks for providing this wonderful tool. Wish it could just work! 😄 

As a Trello power user, I've taken the liberty to fix up something I use all the time — the short URL functionality.

There are some outdated styles that I've now fixed so DOM lookup should work, plus I've made some minor adjustments.

If you're not maintaining this any longer, I'll be happy to do so!

Cheers!

### CODE REVIEW

**Comment by tonytamps** [comment]
Hey mate, thanks for this. These kinds of changes are going to pop up every now and again.
DOM interactions aren't the most reliable way to make shit happen when you don't control the DOM.

We're passively maintaining the repo. There are enough users (many of which are programmers) who help out by submitting PRs or issues. We really appreciate the effort. We also like it this way, which is why it's open source! 

I'll give this a test and make sure this is working on my machine.

File: src/inject/inject.js [hunk 1]
```diff
@@ -9,10 +9,10 @@ chrome.extension.sendMessage({}, function() {
       clearInterval(readyStateCheckInterval);
       chrome.runtime.onMessage.addListener(routeCommand);
     }
-  }, 10);
+  }, 3);
 
   function routeCommand(request) {
-    card = $('.active-card');
+    card = $('.card-detail-window');
```
Comment (tonytamps) [inline src/inject/inject.js]: So this actually breaks one of my favourite pieces of functionality.

I would mouseover a card in the board and press "copy short url". It means I didn't have to wait for cards to load to get their URL for someone else.

Maybe a fallback would be enough? `$('.card-detail-window') || $('.active-card')`

**Review by tonytamps — COMMENTED** [review]
The rest seems fine. As long as the message still appears in a good spot when copying a card on the board

File: src/inject/inject.js (hunk 1 above)
Comment (AeroCross) [reply src/inject/inject.js]: Ah, good one! Let me see if that works because I couldn't find `.active-card` in the DOM when I tried.

Comment (tonytamps) [reply src/inject/inject.js]: It's still there. It's the card that your mouse is currently hovering over. It's how the spacebar shortcut in Trello identifies which card to assign to you.

**Comment by AeroCross** [comment]
Updated, @tonytamps — this should address your concerns. Thanks for taking a look at this!

I will rebase https://github.com/bulkan/shortcuts-for-trello/pull/10 after you're done with this one.

File: src/inject/inject.js (hunk 1 above)
Comment (AeroCross) [reply src/inject/inject.js]: `$('.card-detail-window') || $('.active-card')` didn't work — see https://github.com/bulkan/shortcuts-for-trello/pull/9/commits/43dd575d78a65d115ff8c62bd7cd3ce199af671b for fix.

**Comment by bulkan** [comment]
hey @tonytamps can we merge this now ?

**Comment by tonytamps** [comment]
@bulkan all good with me

**Comment by AeroCross** [comment]
Unscrewed the whole thing in https://github.com/bulkan/shortcuts-for-trello/pull/11 😅

**Pull request closed without merging**